backtracking/generate_parenthesis.py

```diff
@@ -0,0 +1,47 @@
+import doctest
+
+
+def solve(op: str, open: int, close: int, ans: list[str]) -> None:
```

Comment:
follow proper naming convention to choose a valid identifiers, and add doctest too in your code.